### PR TITLE
Build and publish images to github packages

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: hmpps-clamav/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+USER 1000
+
+RUN freshclam --no-dns && \
+    freshclam
+
+VOLUME /var/lib/clamav
+
+COPY --chown=clamav:clamav docker-entrypoint.sh /
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
```
Comment (steverendell) [inline hmpps-clamav/Dockerfile:57]: This seems to run freshclam on startup as well as when building the image, did you intend that or have i misunderstood? The problem we have currently is where clam servers fail to start becuase of the throttling error

Comment (mattops) [reply hmpps-clamav/Dockerfile:57]: Good spot - need to remove that.

Comment (mattops) [reply hmpps-clamav/Dockerfile:57]: Although when we run this image we never use this entrypoint.

Comment (mattops) [reply hmpps-clamav/Dockerfile:57]: We only run `clamd` - see the `CMD` in the daily image - and also the run options on the k8s config

Comment (steverendell) [reply hmpps-clamav/Dockerfile:57]: 👍

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**